src/pages/item/[itemId].tsx

```diff
@@ -36,25 +36,28 @@ const ProductDetails = ({ itemId, metaData }: Props) => {
   const formattedAddress = (address: string) =>
     `${address.substring(0, 7)}...${address.substring(address.length - 4, address.length)} - NFT`;
 
+  const pageTitleWindow = item
+    ? `${item?.name} #${item?.itemId.slice(0, 5)} - Jhon Doe | LEDA`
```

Comment:
Let's just remove the fake data. We are going to prod soon and we will do this clean up anyways